### PR TITLE
Dw 354/discord messages api issue

### PR DESCRIPTION
**Pull request by johnmark13**
Discord message IDs have passed the quintillion mark and this has led to an infinite loop in the codebase due to a string comparison on numeric IDs. This PR proposes a simple fix of casting the IDs to numbers before doing the comparison.

Manually tested and working, thanks to @hozzjss for the work.

### CODE REVIEW

File: packages/sourcecred/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "description": "a tool for communities to measure and reward value creation",
   "homepage": "https://sourcecred.io",
   "repository": "github:sourcecred/sourcecred",
-  "version": "0.11.1",
+  "version": "0.11.2",
```
Comment (hozzjss) [inline packages/sourcecred/package.json:6]: no need to include the version upgrade

File: .github/workflows/yarn.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Yarn CI
```
Comment (hozzjss) [inline .github/workflows/yarn.yml:1]: this whole file might be a different pr

**Review by hozzjss — CHANGES_REQUESTED** [review]

File: packages/sourcecred/package.json (hunk 1 above)
Comment (johnmark13) [reply packages/sourcecred/package.json]: Agree.
